EventEnabledInsurance/post-load-test.sh

```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+#******************************************************************************
+# Licensed Materials - Property of IBM
+# (c) Copyright IBM Corporation 2019. All Rights Reserved.
+#
+# Note to U.S. Government Users Restricted Rights:
+# Use, duplication or disclosure restricted by GSA ADP Schedule
+# Contract with IBM Corp.
+#******************************************************************************
+
+# PREREQUISITES:
+#   - Logged into cluster on the OC CLI (https://docs.openshift.com/container-platform/4.4/cli_reference/openshift_cli/getting-started-cli.html)
+#
+# PARAMETERS:
+#   -n : <NAMESPACE> (string), Defaults to 'cp4i'
+#
+#   With defaults values
+#     ./post-load-test.sh
+#
+#   With overridden values
+#     ./post-load-test.sh -n <NAMESPACE>
+
+function divider() {
+    echo -e "\n-------------------------------------------------------------------------------------------------------------------\n"
+}
+
+function usage() {
+    echo "Usage: $0 -n <NAMESPACE>"
+    divider
+    exit 1
+}
+
+tick="\xE2\x9C\x85"
+cross="\xE2\x9D\x8C"
+all_done="\xF0\x9F\x92\xAF"
+info="\xE2\x84\xB9"
+CURRENT_DIR=$(dirname $0)
+NAMESPACE="cp4i"
+TARGET_POST_CALLS=1000
```

Comment:
1000 sounds like a lot. I guess this takes 15 to 20 mins for DB Writer to catch up and empty the queue? How long does it take for this script to run with 1000?

I'm wondering if maybe we should add a parameter to specify this so it's easy to change.